js/pro/cex.js

```diff
@@ -0,0 +1,1216 @@
+            'oid': this.requestId (),
+        };
+        const request = this.deepExtend (subscribe, params);
+        return await this.watch (url, messageHash, request, messageHash, request);
+    }
+
+    handleBalance (client, message) {
+        //
+        //     {
+        //         e: 'get-balance',
+        //         data: {
+        //             balance: {
+        //                 BTC: '0.00000000',
+        //                 USD: '0.00',
+        //                 ...
+        //             },
+        //             obalance: {
+        //                 BTC: '0.00000000',
+        //                 USD: '0.00',
+        //                 ...
+        //             },
+        //             time: 1663761159605
+        //         },
+        //         oid: 1,
+        //         ok: 'ok'
+        //     }
+        //
+        const data = this.safeValue (message, 'data', {});
+        const freeBalance = this.safeValue (data, 'balance', {});
+        const usedBalance = this.safeValue (data, 'obalance', {});
+        const result = {};
+        const currencyIds = Object.keys (freeBalance);
+        for (let i = 0; i < currencyIds.length; i++) {
+            const currencyId = currencyIds[i];
+            const account = this.account ();
+            account['free'] = this.safeString (freeBalance, currencyId);
+            account['used'] = this.safeString (usedBalance, currencyId);
+            const code = this.safeCurrencyCode (currencyId);
+            result[code] = account;
+        }
+        this.balance = this.safeBalance (result);
+        client.resolve (this.balance, 'balance');
+    }
+
+    async watchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name cex#watchTrades
+         * @description get the list of most recent trades for a particular symbol. Note: can only watch one symbol at a time.
+         * @see https://cex.io/websocket-api#old-pair-room
+         * @param {string} symbol unified symbol of the market to fetch trades for
+         * @param {int|undefined} since timestamp in ms of the earliest trade to fetch
+         * @param {int|undefined} limit the maximum amount of trades to fetch
+         * @param {object} params extra parameters specific to the cex api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/en/latest/manual.html?#public-trades}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        symbol = market['symbol'];
+        const url = this.urls['api']['ws'];
+        const messageHash = 'trades';
+        const subscriptionHash = 'old:' + symbol;
+        const client = this.safeValue (this.clients, url, {});
+        const subscriptions = this.safeValue (client, 'subscriptions', {});
+        const subscriptionKeys = Object.keys (subscriptions);
+        for (let i = 0; i < subscriptionKeys.length; i++) {
```

Comment:
@pcriadoperez this is breaking in python can you take a look?
![image](https://user-images.githubusercontent.com/43336371/197577786-952908bc-ceeb-4b7e-a2bc-6b1322d80855.png)